data.js

```diff
@@ -739,4 +739,44 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2019-12-21"
         },
     },
+    {
+        meta: {
+            name: "Climate Reality Project",
+            link: "https://www.climaterealityproject.org/",
+            label: "climate-realtiy-project",
+        },
+        description: {
+            content: "Founded by former Vice President Al Gore, the Climate Reality works to establish a global network of climate activists by regular trainings and supporting them in spreading the word.",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },
+        established: {
+            content: "(2006)[https://www.climaterealityproject.org/whoweare]",
+            updatedBy: "Andre", updatedAt: "2020-01-20"
+        },   
+        kpis: {
+                content: "- presentations / pa \n- newly trained climate reality leaders (people able to hold talks and spread the word) / pa \n- downloaded resources / pa",
+                updatedBy: "Andre", updatedAt: "2020-01-20",
+        },
+        actions: {
+            content: "- holding presentations in 150 countries; individuals can (request)[https://realityhub.climaterealityproject.org/events/attendpresentation] a speaker \n- holding up team meetings around the US \n- and other (initiatives)[https://climaterealityproject.org/initiative] like a collaboration with the musician Jack Johnson",
```

Comment:
@yeah800 I didn't double-check thoroughly, sorry. Most hyperlinks in this PR are wrong. Please fix